assets/pages/oneoff-contributions/components/oneoffInlineContributionsPayment.jsx

```diff
@@ -0,0 +1,178 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
+
+import StripeInlineForm from 'components/stripeInlineForm/stripeInlineForm';
+import PayPalContributionButton from 'containerisableComponents/payPalContributionButton/payPalContributionButton';
+import ErrorMessage from 'components/errorMessage/errorMessage';
+
+import { validateEmailAddress } from 'helpers/utilities';
+
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { ReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
+import type { Participations } from 'helpers/abTests/abtest';
+import type { Currency } from 'helpers/internationalisation/currency';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import type { Status } from 'helpers/switch';
+import { stripeInlineFormActionsFor } from 'components/stripeInlineForm/stripeInlineFormActions';
+
+import { checkoutError, type Action } from '../oneoffContributionsActions';
+import postCheckout from '../helpers/ajax';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  dispatch: Function,
```

Comment:
`Dispatch`?

Reply:
Addressed in ee3700ce0a0adb3617e2ef7a5a84c28bf59b0272